src/main/java/com/ays/backend/mapper/UserMapper.java

```diff
@@ -1,13 +1,23 @@
 package com.ays.backend.mapper;
 
+import com.ays.backend.user.controller.payload.request.UpdateUserRequest;
 import com.ays.backend.user.model.entities.User;
 import com.ays.backend.user.service.dto.UserDTO;
+import org.mapstruct.AfterMapping;
 import org.mapstruct.Mapper;
+import org.mapstruct.Mapping;
+import org.mapstruct.MappingTarget;
+
+import java.time.LocalDateTime;
 
 @Mapper(componentModel = "spring")
```

Comment:
please change to:
@Mapper(componentModel = "spring", nullValuePropertyMappingStrategy = NullValuePropertyMappingStrategy.IGNORE)